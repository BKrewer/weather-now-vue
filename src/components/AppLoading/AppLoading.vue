<template>
  <div class="app-loading">
    <img class="app-loading__loader" alt="Loading" src="@/assets/images/loader.svg">
  </div>
</template>

<style lang="scss" src="./AppLoading.scss" />