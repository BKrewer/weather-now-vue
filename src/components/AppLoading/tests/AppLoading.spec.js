import { shallowMount } from '@vue/test-utils'
import AppLoading from '@/components/AppLoading/AppLoading.vue'

describe('AppLoading.vue', () => {
  it('renders correctly', () => {
    const wrapper = shallowMount(AppLoading)
    expect(wrapper.element).toMatchSnapshot();
  })
})