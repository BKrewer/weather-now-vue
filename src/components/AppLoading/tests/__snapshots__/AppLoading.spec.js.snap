// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppLoading.vue renders correctly 1`] = `
<div
  class="app-loading"
>
  <img
    alt="Loading"
    class="app-loading__loader"
    src=""
  />
</div>
`;
